### PR TITLE
Add full name composite field

### PR DESCRIPTION
**Pull request by thomtrp**
- Add composite field
- Add test
- Fix search variable dropdown scroll
Capture d’écran 2024-12-10 à 16 52 43

### CODE REVIEW

File: packages/twenty-front/src/modules/object-record/record-field/form-types/components/StyledFormCompositeFieldInputContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  align-self: stretch;
+  background: ${({ theme }) => theme.background.secondary};
+  border: 1px solid ${({ theme }) => theme.border.color.medium};
+  border-radius: ${({ theme }) => theme.border.radius.sm};
+  gap: ${({ theme }) => theme.spacing(2)};
+  padding: ${({ theme }) => theme.spacing(2)};
+`;
```
Comment (martmull) [inline packages/twenty-front/src/modules/object-record/record-field/form-types/components/StyledFormCompositeFieldInputContainer.tsx:13]: Duplicated with `StyledInputContainer` in `renderFields` in `packages/twenty-front/src/modules/workflow/workflow-actions/components/WorkflowEditActionFormServerlessFunction.tsx`
Think we should factorize

File: packages/twenty-front/src/modules/object-record/record-field/meta-types/input/constants/FirstNamePlaceholder.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+export const FIRST_NAME_PLACEHOLDER_WITH_SPECIAL_CHARACTER_TO_AVOID_PASSWORD_MANAGERS =
+  'F‌‌irst name';
```
Comment (martmull) [inline packages/twenty-front/src/modules/object-record/record-field/meta-types/input/constants/FirstNamePlaceholder.ts:2]: jeez 😅

**Review by martmull — APPROVED** [review]
LGTM, one small comment

**Review by Devessier — APPROVED** [review]
It looks great to me 👌 It works perfectly.

File: packages/twenty-front/src/modules/object-record/record-field/form-types/components/StyledFormCompositeFieldInputContainer.tsx (hunk 1 above)
Comment (thomtrp) [reply packages/twenty-front/src/modules/object-record/record-field/form-types/components/StyledFormCompositeFieldInputContainer.tsx:13]: Good catch!

**Pull request merged**